### PR TITLE
AQI-5841 added spreadsheet parameter biding with historian data sourc…

### PR DESCRIPTION
**Pull request by AlanLiu-AI**
…e from ingest.

### CODE REVIEW

File: proto/one/common/core/enum_datasource.proto [hunk 1]
```diff
@@ -14,4 +14,5 @@ enum EnumDataSource {
     DATASOURCE_SAMPLE_MANAGER = 8;          // Data related to Sample Manager
     DATASOURCE_WIMS_SYNC = 9;               // Data related to WIMS Sync
     DATASOURCE_WIMS_COMPUTATION = 10;       // Data related to WIMS Computations
+    DATASOURCE_HISTORIAN_DATA_INGEST = 11;  // OP Historian Data collected by OP Ingest
```
[the remark this answers is not in the document]
Comment (AlanLiu-AI) [reply proto/one/common/core/enum_datasource.proto:17]: OP stands for One Platform, I'll switch to full words.

File: proto/one/operations/spreadsheet/data_source_binding.proto [hunk 2]
```diff
@@ -9,4 +11,8 @@ message DataSourceBinding {
     string bindingId = 1;
     EnumDataSource dataSource = 2;
     EnumSamplingStatistic enumSamplingStatistic = 3;
+
+    // Delay seconds addition to calculate latest statistic value. 
+    // E.g. 300 indicates midnight plus 5 minutes to calculate latest daily value.
+    google.protobuf.UInt32Value delaySecondsOnCalcLatestStats = 4;
```
[the remark this answers is not in the document]
Comment (AlanLiu-AI) [reply proto/one/operations/spreadsheet/data_source_binding.proto:17]: Good to know, will change it.

[the remark this answers is not in the document]
Comment (AlanLiu-AI) [reply proto/one/operations/spreadsheet/data_source_binding.proto:17]: As agreed on ADR, this delay will be removed.

**Comment by AlanLiu-AI** [comment]
@AQI-DanG , the delay field has been renamed, and the new enum also renamed according to Martin's point.

**Review by tomsmith-ai — APPROVED** [review]

**Review by AQI-DanG — APPROVED** [review]

**Pull request merged**